### PR TITLE
Improvementes to the SCC documentation

### PR DESCRIPTION
**Pull request by php-coder**
Based on the question about capabilities and SCC, I did an attempt to improve our documentation and make some points clearer.

PTAL @pweil- @smarterclayton 
CC @mfojtik 
JFYI @brenton

### CODE REVIEW

**Comment by smarterclayton** [comment]
Looks ok to me technically.

**Review by pweil- — APPROVED** [review]

**Comment by php-coder** [comment]
PTAL @openshift/team-documentation

**Comment by php-coder** [comment]
@openshift/team-documentation ping

**Comment by ahardin-rh** [comment]
@php-coder Thanks! Overall, this looks good to me. I will just make some very minor style edits in a separate follow-up PR.  First:

1. Can you please squash this into one commit?
2. Is this targeting 3.6?

File: architecture/additional_concepts/authorization.adoc [hunk 1]
```diff
@@ -378,33 +378,38 @@ metadata:
...
 - system:serviceaccount:default:router
 - system:serviceaccount:openshift-infra:build-controller
 volumes:
 - '*'
 ----
 
-<1> The `*FSGroup*` strategy which dictates the allowable values for the
+<1> A list of capabilities that can be requested by a pod. An empty list means
+that none of capabilities can be requested while the special symbol `***`
+allows any capabilities
```
Comment (php-coder) [inline architecture/additional_concepts/authorization.adoc]: @ahardin-rh The only change that is specific to 3.6 is the usage of `*` in `allowedCapabilities`: https://github.com/openshift/origin/pull/12875#issuecomment-299547514

**Comment by php-coder** [comment]
@ahardin-rh Squashed.

> I will just make some very minor style edits in a separate follow-up PR.

I'm curious why documentation team using the follow-up PRs instead of requesting comments to the original PR?

Anyway, could I ask you to always add me to CC to the follow-up PRs of my PRs? I'd like to to learn from my mistakes and also to take best documentation practices from your team.

**Comment by ahardin-rh** [comment]
@php-coder Thanks! We tend to do quick follow-up PRs for simple style edits just to save you time. 

I will be sure to tag you in the follow-up PR. :beers:

**Comment by ahardin-rh** [comment]
No revision history needed

**Pull request merged**